Source/iOS/OIDExternalUserAgentIOS.h

```diff
@@ -27,6 +27,7 @@ NS_ASSUME_NONNULL_BEGIN
 /*! @brief An iOS specific external user-agent that uses the best possible user-agent available
         depending on the version of iOS to present the request.
  */
+API_UNAVAILABLE(macCatalyst)
```

Comment:
Finally, I replaced the #if target statement by this api availability macro. It provides a clearer message to the developer, but still cannot suggest a replacement. macCatalyst being new, this implies AppAuth will only be compatible with Xcode 11+, which is our policy I think.

Reply:
This is great!

Yes, that's our policy. We support old versions of the iOS SDK, but not Xcode (beyond a few months during the update).